src/toil/test/src/helloWorldTest.py

```diff
@@ -19,7 +19,7 @@
 class HelloWorldTest(ToilTest):
     def testHelloWorld(self):
         options = Job.Runner.getDefaultOptions(self._getTestJobStorePath())
-        options.logLevel = "INFO"
+        options.logLevel = "DEBUG"
```

Comment:
I think the test should just use the default value (i.e. delete this line)

Reply:
I like leaving it in, but `info` or `critical` will reduce the logs it produces.  Don't set this to `debug`.